mender-deb-package

```diff
@@ -97,7 +97,7 @@ get_deb_version() {
   #  - For master: <debian-epoch>X.Y.Z~git<commit-date>.<commit-sha>-<debian_suffix>+b<BUILD_ID>
   #     where X.Y.Z is latest tag (not necessarily matching git describe)
   debian_epoch=""
-  if [ "${DEB_PACKAGE}" = "mender-client" ]; then
+  if [[ "${DEB_PACKAGE}" =~ ^mender-client(4)?$ ]]; then
     debian_epoch="1:"
```

Comment:
So we continue mender-client4 at epoch 1. Can't we keep that only for mender-client? My impression is that it should be used sparingly.

Reply:
Mmm. I first thought that we had to keep it in order to upgrade but now I see that it doesn't make sense: it is a completely new package.

If we ever want to introduce a `mender-client` meta-package version 4 to force the upgrade then we need to have the epoch there, but as you are pointing out that doesn't imply that we need the epoch in `mender-client4`.

Thanks! I'll remove it.